src/Tribe/Views/V2/Views/Traits/List_Behavior.php

```diff
@@ -91,11 +91,21 @@ static function ( $a, $b )
 				$start = $first_event instanceof \WP_Post ? $first_event->dates->start_display : $user_date;
 			}
 			$end = $last_event instanceof \WP_Post ? $last_event->dates->start_display : $user_date;
+
+			// never let the start of the range be lower than the top bar date
+			if ( $start < $user_date ) {
+				$start = $user_date;
+			}
 		} else {
 			$start = $first_event instanceof \WP_Post ? $first_event->dates->start_display : $user_date;
 			$end   = $last_event instanceof \WP_Post ? $last_event->dates->start_display : $user_date;
 		}
 
+		// never let the start of the range exceed the start
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 61bb0ee9fa9444dc387876230c364f70</sub>